### DISCUSSION TITLE
how can i use emailjs in firebase cloud function

### DISCUSSION THREAD

**Discussion by iftekharAi** [emailjs-com/help#68] [Q&A]
i am unable to use emailjs in firebase cloud function although i can use their rest api but it has rate limiter but i have to sent n number of email in one call   .... any idea how to use this..... can i use here emailjs sdk if yes please tell me from where i can use sdk

**Comment by xr0master** [comment]
It’s not entirely clear what you would like to achieve, what is your goal?

In general, the service is not built for mass emailing; this service is for sending emails by a trigger. However, different options are possible.
